codeclimate.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.5"
 
-  s.add_dependency "activesupport", ">= 5.1.3", "< 5.2"
+  s.add_dependency "activesupport", ">= 5.2"
```

Comment:
sorry to bring this again, I think this should use the `Pessimistic Version Operator` (`~>`) like the others below.
Maybe something like `~> 5.2.4`, allowing it to apply new patches but without jumping minor versions, that might be problematic with activesupport I believe. WDYT?

Reply:
Actually that sounds like a clever idea. Done.

> sorry to bring this again
No, thanks for always find a way to improve the work.